tools/android_lint/baseline.xml

```diff
@@ -30,58 +30,69 @@
         errorLine2="               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~">
         <location
             file="../../../flutter/shell/platform/android/io/flutter/embedding/engine/systemchannels/PlatformChannel.java"
-            line="229"
+            line="275"
             column="16"/>
     </issue>
 
     <issue
         id="InlinedApi"
-        message="Field requires API level 21 (current min is 16): `android.view.HapticFeedbackConstants#CLOCK_TICK`"
-        errorLine1="                view.performHapticFeedback(HapticFeedbackConstants.CLOCK_TICK);"
-        errorLine2="                                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~">
+        message="Field requires API level 21 (current min is 16): `android.view.WindowManager.LayoutParams#FLAG_DRAWS_SYSTEM_BAR_BACKGROUNDS`"
```

Comment:
can this be removed if we specify a `TargetAPI` annotation in ?https://github.com/flutter/engine/blob/143e32a7672e301c6d6ab7ad63ab607a57c3d5fd/shell/platform/android/io/flutter/plugin/platform/PlatformPlugin.java#L366

Reply:
similar comment for the ones below, but using the specified min API